pages/home.js

```diff
@@ -92,79 +107,87 @@ export default function Home(props) {
+            value={surname}
+            onChange={setSurname}
+            errorMessage={surnameError}
+          />
+          <InputFeild
+            id="dob"
+            name="birthDate"
+            label={props.content.birthDate.label}
+            required
+            textRequired={props.commonContent.required}
+            value={birthDate}
+            onChange={setBirthDate}
+            errorMessage={birthDateError}
+            type="date"
+          />
+          <ActionButton
+            type="submit"
+            text={props.content.checkStatus}
+            style="primary"
+          />
+        </form>
+      ) : (
         <div id="response">
           {response.success ? (
-            <p className="border-l-6 border-accent-info mb-6 ml-2.5 pl-4 text-2xl">
+            <p className="mb-6 text-2xl">
               {props.content.statusIs}{' '}
               <strong>{props.content.status[response.status]}</strong>.
             </p>
           ) : (
-            <p className="border-l-6 border-accent-warning mb-6 ml-2.5 pl-4 text-2xl">
-              {props.content.unableToFindStatus}
-            </p>
+            <p className=" mb-6 text-2xl">{props.content.unableToFindStatus}</p>
           )}
         </div>
       )}
-
-      {!errorSummary ? null : (
-        <ErrorSummary
-          id="error-summary-get-status"
-          summary={props.commonContent.foundErrors}
-          errors={errorSummary}
-        />
-      )}
-      <form onSubmit={handleSubmit} id="form-get-status">
-        <InputFeild
-          id="esrf"
-          name="FileNumber"
-          label={props.content.esrf.label}
-          required
-          textRequired={props.commonContent.required}
-          value={esrf}
-          onChange={setEsrf}
-          errorMessage={esrfError}
-        />
-        <InputFeild
-          id="givenName"
-          name="givenName"
-          label={props.content.givenName.label}
-          required
-          textRequired={props.commonContent.required}
-          value={givenName}
-          onChange={setGivenName}
-          errorMessage={givenNameError}
-        />
-        <InputFeild
-          id="surname"
-          name="surname"
-          label={props.content.surname.label}
-          required
-          textRequired={props.commonContent.required}
-          value={surname}
-          onChange={setSurname}
-          errorMessage={surnameError}
-        />
-        <InputFeild
-          id="dob"
-          name="birthDate"
-          label={props.content.birthDate.label}
-          required
-          textRequired={props.commonContent.required}
-          value={birthDate}
-          onChange={setBirthDate}
-          errorMessage={birthDateError}
-          type="date"
-        />
-        <ActionButton
-          type="submit"
-          text={props.content.checkStatus}
-          style="primary"
-        />
-      </form>
+      {response ? (
+        <div>
```

Comment:
Again, we could move this up, put it under the same condition use to show the response